### DISCUSSION TITLE
The question is in the issues section

### DISCUSSION THREAD

**Discussion by paperboy6891** [paperboy6891/Final-Project#3] [Q&A] [answered]
You can look at the question in the issues section for info

**Comment by arhmAli** [comment, marked as answer]
I have atttached the solution its little things that matters right?
So The things which i think will be a good start for you is 

- Convert the regular function expression to arrow function.
- Convert the vars to let just QOL improvement
- The main thing which effects the performance of your code is not using for loop in these cases you can use a lot of arrays methods like reduce and forEach which will help the O(n) to get better and compilation time will be much lowered!
